resources/operator.yaml

```diff
@@ -70,7 +70,7 @@ spec:
             - containerPort: 6379
           resources:
             limits:
-              cpu: "0.1"
+              cpu: "2.0"
```

Comment:
This is a pretty big change... Any specific reason ?

Reply:
When running scale, I found that each skylake core used by redis-server can sustain about 1500 redis ops per second. We had runs with 200 nodes x 16 density = 3200 client pods. Hence it needs 2 cores. Besides upping the cpu to 2, I also put 0.5 sec delays in the client's while loop that spins and pulls redis for 'start' signal.